tensorflow/contrib/seastar/seastar_server_lib.cc

```diff
@@ -0,0 +1,522 @@
+#include <fstream>
+#include <map>
+#include <stdexcept>
```

Comment:
I am not sure about exceptions here. Better changing it to ``tensorflow::Status``.